pkg/image/api/helper_test.go

```diff
@@ -291,8 +291,16 @@ func TestParseDockerImageReference(t *testing.T) {
 			Err:  true,
 		},
 		{
-			From: "bar/foo/baz/biz",
-			Err:  true,
+			From:      "bar/foo/baz/biz",
+			Registry:  "bar",
```

Comment:
That's right. See my above comment https://github.com/openshift/origin/pull/11173/files#r81712056